config.go

```diff
@@ -241,7 +264,27 @@ func ValidateEnvs(envs Envs) (errors []string) {
 							i, *env.Name, *env.Value))
 				}
 			}
+		}
+	}
+
+	return
+}
+
+// validateOptions checks that input Options are correctly set.
+// Returns array of error messages, empty if none errors are found
+func validateOptions(options Options) (errors []string) {
```

Comment:
Why no validation for the other options? Seems like you could check them, for example, similar to this.

- `minScale >= 0`
- `maxScale >= minScale`
- `initialScale >= minScale`
- `0s <= window <= 1h`
- `utilization <= 100`